cache/src/main/scala/coursier/Cache.scala

```diff
@@ -1134,6 +1166,7 @@ object Cache {
     def downloadedArtifact(url: String, success: Boolean): Unit = {}
     def checkingUpdates(url: String, currentTimeOpt: Option[Long]): Unit = {}
     def checkingUpdatesResult(url: String, currentTimeOpt: Option[Long], remoteTimeOpt: Option[Long]): Unit = {}
+    def log(content: String, currentTimeOpt: Option[Long]): Unit = {}
```

Comment:
I think it'd be better to have a more specific method for the error we're reporting… like
```scala
def removedCorruptFile(url: String, file: File, reason: Option[FileError]): Unit
```

Reply:
Done